packages/realm/src/platform/node/https-proxy.ts

```diff
@@ -0,0 +1,49 @@
+////////////////////////////////////////////////////////////////////////////
```

Comment:
I suggest renaming this file to `sync-proxy-config.ts` and it needs to be imported from `node/index.ts` to include it in the bundle.